go.mod

```diff
@@ -5,17 +5,23 @@ go 1.13
 require (
 	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible
-	github.com/Microsoft/hcsshim v0.8.7
+	github.com/Microsoft/go-winio v0.5.0 // indirect
+	github.com/Microsoft/hcsshim v0.8.20
+	github.com/Microsoft/hcsshim/test v0.0.0-20210726222751-a8f5f611a659 // indirect
 	github.com/asaskevich/govalidator v0.0.0-20200428143746-21a406dcc535
 	github.com/avast/retry-go v2.6.0+incompatible
 	github.com/cloudflare/cfssl v1.4.1
-	github.com/containerd/containerd v1.4.1
+	github.com/containerd/containerd v1.5.4
```

Comment:
I wonder if we should use containerd 1.4.9 instead

Reply:
i think it is helm 3.6.0 that pulls in containerd 1.5.4

the help update is also breaking the windows build